### PR TITLE
[NUI] Constructor of NUIWidgetApplication is opened public in API level7

### PR DESCRIPTION
**Pull request by tscholb**
For apply multiple class of Widget, this API need to open public

### API Changes ###
https://code.sec.samsung.net/jira/browse/TCSACR-299

### CODE REVIEW

**Comment by tscholb** [comment]
Add ACR link : TCSACR-299

**Comment by dongsug-song** [comment]
@tscholb 
Hello, please submit this PR in master branch also.

**Comment by dongsug-song** [comment]
@tscholb 
Hello,
currently, TCSACR-299 looks under progress. could you update and handle the ACR?
this can be merged after the ACR is accepted.
thank you.

File: src/Tizen.NUI/src/public/NUIWidgetApplication.cs [hunk 1]
```diff
@@ -42,7 +42,6 @@ public NUIWidgetApplication( System.Type widgetType ) : base(new NUIWidgetCoreBa
         /// The constructor for multi widget class and instance.
         /// </summary>
         /// <param name="widgetTypes">List of derived widget class type.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public NUIWidgetApplication(Dictionary<System.Type, string> widgetTypes) : base(new NUIWidgetCoreBackend())
```
Comment (hyunjushin) [inline src/Tizen.NUI/src/public/NUIWidgetApplication.cs:45]: is this constructor to support multiple widget class? Not clear the title & description of this patch.

**Comment by dongsug-song** [comment]
ACR is accepted, so it is merged

**Pull request merged**